app/api/services/abr.py

```diff
@@ -72,6 +73,7 @@ def get_abr_exception(self, response):
             return None
 
     def get_data(self, response):
+        response = response.decode()
```

Comment:
I think it might be better to `decode()` where these functions are called.

Reply:
Resolved